src/cli/cli.cpp

```diff
@@ -674,6 +674,29 @@ void Interpreter::ProcessBufferInfo(uint8_t aArgsLength, char *aArgs[])
     OutputResult(OT_ERROR_NONE);
 }
 
+void Interpreter::ProcessCcaThreshold(uint8_t aArgsLength, char *aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgsLength == 0)
+    {
+        int8_t cca;
+
+        SuccessOrExit(error = otPlatRadioGetCcaEnergyDetectThreshold(mInstance, &cca));
+        OutputLine("%d dBm", cca);
+    }
+    else
+    {
+        long value;
+
+        SuccessOrExit(error = ParseLong(aArgs[0], value));
```

Comment:
Validate the value range of `value`?

Reply:
Yes, I modeled this exactly as the `ProcessTxPower` (it's almost a copy & paste):
https://github.com/openthread/openthread/blob/b735193857cdbd490cb71f274c9794e4b8995e39/src/cli/cli.cpp#L3760-L3765

If you want I can change both methods to validate this cast/truncation.

Reply:
Fine to change both or stay in the scope of this PR.

Reply:
Ok, if you agree we can merge this as it is, and I'll open an issue to address the range validation in `ccathreshold` and `txpower` (and possibly others?).